prow/logrusutil/logrusutil.go

```diff
@@ -27,39 +27,46 @@ import (
 type DefaultFieldsFormatter struct {
 	WrappedFormatter logrus.Formatter
 	DefaultFields    logrus.Fields
+	PrintLineNumber  bool
 }
 
-// NewDefaultFieldsFormatter returns a DefaultFieldsFormatter,
-// if wrappedFormatter is nil &logrus.JSONFormatter{} will be used instead
-func NewDefaultFieldsFormatter(
-	wrappedFormatter logrus.Formatter, defaultFields logrus.Fields,
-) *DefaultFieldsFormatter {
-	res := &DefaultFieldsFormatter{
-		WrappedFormatter: wrappedFormatter,
-		DefaultFields:    defaultFields,
+// Init set Logrus formatter
+// if DefaultFieldsFormatter.wrappedFormatter is nil &logrus.JSONFormatter{} will be used instead
+func Init(formatter *DefaultFieldsFormatter) {
```

Comment:
Should we check that `formatter` is not null?

Reply:
@ibrasho Good catch, can you please check?